.github/workflows/ci.yml

```diff
@@ -135,3 +135,11 @@ jobs:
         run: |
           php --version
           ./build.php $env:AUTOLOAD
+
+      - name: Coveralls
```

Comment:
nit: This is the third pattern of running coveralls in addition to a separate job and the parallel matrix with finish job. I think it's the best one but we need to look into consistency a bit.

Reply:
I agree, I plan to stick with this pattern and go back to change the rest of the libraries to follow it.